adonis-typings/i18n.ts

```diff
@@ -185,6 +185,11 @@ declare module '@ioc:Adonis/Addons/I18n' {
      */
     provideValidatorMessages: boolean
 
+    /**
+     * Set this to true when you want to return the translation key as a fallback.
+     */
+    returnKeyAsFallback: boolean
```

Comment:
Should be an optional config property with a default value of false, otherwise, it represents a breaking change. People will get typing errors for their config after upgrading

Reply:
This should all be resolved in the latest commit.